python/GafferUI/ExpressionUI.py

```diff
@@ -174,6 +175,46 @@ def __init__( self, node, **kw ) :
 
 		self.__update()
 
+	def node( self ) :
+
+		return self.__node
+
+	def textWidget( self ) :
+
+		return self.__textWidget
+
+	__popupMenuSignal = Gaffer.Signal2()
+	## This signal is emitted whenever a popup menu for a plug is about
```

Comment:
you spotted my copy/paste magic... fixed in fbd419fc5120c55421dadec7a624bb28ec051ccc